app/views/billing_expenses/show.html.slim

```diff
@@ -79,3 +79,61 @@
             = t_attr(:created_by)
           .col-xs-6
             = @billing_expense.user.profile.contact.full_name
+
+.row
+  br
+  br
+  br
+  br
+  br
+  .col-xs-7.col-xs-offset-1
+    .table-responsive
```

Comment:
What is the need of a responsive table in something you intend to put on a pdf?

Reply:
With show this renders at the browser. The pdf is produced when we press download.